src/db/schema.rs

```diff
@@ -16,17 +16,17 @@ table! {
     /// The crate metadata table.
     crates (id) {
         /// The crate's ID.
-        id -> Unsigned<Bigint>,
+        id -> Bigint,
         /// The crate's name.
         name -> Varchar,
         /// The crate's descripton.
         description -> Nullable<Varchar>,
         /// The crate's creation date.
-        created_at -> Datetime,
+        created_at -> Varchar,
```

Comment:
Yes, while searching for commonly supported datatypes for these columns, I considered using `Timestamp`.  
The reason I ended up not using it is that I find the bounds that MySQL uses for its `TIMESTAMP` type to be too short.  
They go from '1970-01-01 00:00:01' UTC to '2038-01-19 03:14:07' UTC according to this page: https://dev.mysql.com/doc/refman/8.0/en/datetime.html.
I also thought it could make reading dates directly from the database easier for humans.